Actions/IncrementVersionNumber/IncrementVersionNumber.psm1

```diff
@@ -0,0 +1,94 @@
+﻿<#
+    .Synopsis
+        Changes a setting value in a settings file.
+    .Parameter settingsFilePath
+        Path to the settings file.
+    .Parameter settingName
+        Name of the setting to change.
+    .Parameter newValue
+        New value of the setting.
+    .Parameter incremental
+        If set, the new value will be added to the old value. The old value must be a version number.
+#>
+function Set-SettingInFile($settingsFilePath, $settingName, $newValue, [switch] $incremental) {
```

Comment:
These are not specific for the action. Move to a common helper module? Or AL-Go-Helper.ps1.

Reply:
Mmh Set-SettingInFile could be generally useful. The Incremental switch seems fairly specific to this action though 😄  If you could refactor that somehow then I agree

Reply:
I think the name should be more like `Set-VersionSetting` because it's not as generic.
Then `$incremental` would make sense.

Reply:
Changed the function name to `Set-VersionSettingInFile` to handle only version settings.

Also, included the "incremental" logic in the function itself.